guardian/prosemirror-invisibles
# Bump prosemirror

## What does this change?

Bumps prosemirror dependencies to their latest, including some housekeeping –

- the newest versions of Prosemirror are written in Typescript and have their own types, so we must make some adjustments to fit.
- our bundle process was including our prosemirror dependencies in the CJS build as a result of #48. This PR removes them by moving them to `peerDependencies` and explicitly excluding them. 

## How to test

Run locally. Does the project work as expected?

Import into a consuming project, perhaps with `yalc`. Does the project work as expected?
